spec/models/publication_spec.rb

```diff
@@ -473,6 +481,11 @@
       pub = described_class.new(pub_hash: { provenance: 'wos' })
       expect { pub.rebuild_pub_hash }.to raise_error(RuntimeError)
     end
+
+    it 'raises for cap record' do
+      pub = described_class.new(pub_hash: { provenance: 'cap' })
+      expect { pub.rebuild_pub_hash }.to raise_error(RuntimeError)
+    end
   end
```

Comment:
some new behavior, we should also raise an error for other types that don't have rebuilding the pubhash supported